packages/rest/src/rest.component.ts

```diff
@@ -37,6 +49,21 @@ export class RestComponent implements Component {
     [RestBindings.SequenceActions.PARSE_PARAMS.key]: ParseParamsProvider,
     [RestBindings.SequenceActions.SEND.key]: SendProvider,
   };
+  /**
+   * Add built-in body parsers
+   */
+  bindings = [
+    Binding.bind(RestBindings.REQUEST_BODY_PARSER)
+      .toClass(RequestBodyParser)
+      .inScope(BindingScope.SINGLETON),
+    createBodyParserBinding(JsonBodyParser).inScope(BindingScope.SINGLETON),
```

Comment:
IMO, body parsers should be singletons by default and thus `createBodyParserBinding` should be setting the scope for all bindings it creates.

What use cases do you see for non-singleton parsers? 

Even when we change `createBodyParserBinding` to set the scope to SINGLETON, non-singleton parsers can still override it:

```ts
createBodyParserBinding(TransientParser).inScope(BindingScope.TRANSIENT)
```